### PR TITLE
Fix default winsize

### LINKED ISSUE #77: aggregate adding files to METS to gain speed
Currently, for each page, after the backend CLI have created the output image and output PAGE, we add both separately to the METS.

Since METS de/serialization may become very slow on large workspaces, it would be better to aggregate all these `ocrd workspace add` commands in the loop into a single `ocrd workspace bulk-add` in the end. (But of course, we would have to be careful with error handling.)

### LINKED ISSUE #84: default parameterization of wolf is suboptimal
I have 600 DPI images that look pretty harmless, e.g.

0357_Page 354

But when fed to our default parameterization for `wolf`, they fail utterly:

win-size-601

It turns out that our rule of thumb `win-size=DPI` chosen with considerations based on the Sauvola formula does not work with Wolf&Jolion(2004)'s algorithm. Here's the same image with the default window size of 101 (instead of 601):

win-size-101

The same seems to be true for `niblack`, `singh` and `kim` (although not as extreme).

I therefore suggest changing the default to `win-size=DPI/6` for those algorithms.

### PR DESCRIPTION
**Pull request by bertsky**
Fixes #84

Fixes #77

### CODE REVIEW

**Comment by bertsky** [comment]
CI failure is weird. I get a result from `pkg-config` directly after the `deps-ubuntu` call to `apt-get` (i.e. without output from the latter):
```
No package 'Magick++-im6' found
```

Also seeing 404s for apt-get later...

File: ocrd-olena-binarize [hunk 1]
```diff
@@ -386,8 +405,19 @@ function main {
...
+    mkfifo $FIFO
+    trap backout ERR
+    
+    bulk_options=( -r '(?P<grp>[^ ]+) (?P<mime>[^ ]+) (?P<page>[^ ]+) (?P<file>[^ ]+) (?P<url>.*)')
+    bulk_options+=( -G '{{ grp }}' -m '{{ mime }}' -g '{{ page }}' -i '{{ file }}' -S '{{ url }}')
+    if [[ "${ocrd__argv[overwrite]}" == true ]];then
+        bulk_options+=( --force )
+    fi
+    ocrd workspace -m "${mets_basename}" bulk-add "${bulk_options[@]}" - <$FIFO &
+    exec 3>$FIFO
```
Comment (kba) [inline ocrd-olena-binarize:419]: Very clever pattern.

**Review by kba — COMMENTED** [review]
Thanks for the fix and `ocrd workspace {,bulk-}add` improvement. I'll try to debug the CI failure.

**Comment by kba** [comment]
I think I fixed the CI, cf. https://app.circleci.com/pipelines/github/OCR-D/ocrd_olena/165/workflows/366b42a8-d6c9-4d53-990c-3580d4830d98/jobs/165 in branch https://github.com/OCR-D/ocrd_olena/tree/circleci-update-resource-class with a combination of `apt-get update`, updating the base image and guarding against the git `safe.directory` problem.

**Pull request merged**